lib/subjects/selector.rb

```diff
@@ -35,7 +35,16 @@ def active_subjects_in_selection_order(sms_ids)
 
     def fallback_selection
       opts = { limit: subjects_page_size, subject_set_id: subject_set_id }
-      PostgresqlSelection.new(workflow, user, opts).any_workflow_data
+      fallback_selector = PostgresqlSelection.new(workflow, user, opts)
+
+      sms_ids = []
+      sms_ids = fallback_selector.select if workflow.using_cellect?
```

Comment:
Yeah, but it'll be hard not to just start swamping Cellect with reloads near the actual end of a project.